library/manageiq.py

```diff
@@ -214,9 +212,9 @@ def main():
             name=dict(required=True),
             type=dict(required=True,
                       choices=['openshift-origin', 'openshift-enterprise']),
-            url=dict(required=True),
-            username=dict(required=True),
-            password=dict(required=True, no_log=True),
+            url=dict(default=os.environ.get('MIQ_URL', None)),
+            username=dict(default=os.environ.get('MIQ_USERNAME', None)),
+            password=dict(default=os.environ.get('MIQ_PASSWORD', None)),
```

Comment:
> I imagine this call in the constructor to ManageIQ would throw an exception to the fact that credentials were invalid or not found.
>  self.client = MiqApi(self.api_url, (self.user, self.password))

@kwoodson Unfortunately, this validation isn't part of the api client constructor.
I added a separate validation for these 3 credentials.